docs/slurm-perf-cluster.md

```diff
@@ -0,0 +1,182 @@
+SlurmPerf Deployment Guide
+===
+
+## High-Performance Multi-Node Cluster Deployment Guide
+
+   This guide utilizes a collection of playbooks to configure a cluster, deploy a workload manager, and verify performance.
+
+   It leverages open source tools such as Pyxis and Enroot to optimize Slurm for multi-node Deep Learning jobs beyond the cluster configuration described in the [Slurm Deployment Guide](https://github.com/NVIDIA/deepops/blob/master/docs/slurm-cluster.md). Additional details can be found [here](https://docs.nvidia.com/ngc/multi-node-bert-user-guide).
```

Comment:
We should not hardcode the repo/branch here for the cluster doc.

Reply:
Removed hardcoded references (such as this).